ShortcutsSwift.playgroundbook/Contents/Sources/InterpolatedText.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-public func withInterpolatedText(_ string: String) -> Any {
+func withInterpolatedText(_ string: String) -> Any {
```

Comment:
I made this public for a bit to facilitate the creation of Actions in the playground itself, which is easier than doing it in the framework files.